src/coponents/CheckInNav.tsx

```diff
@@ -0,0 +1,72 @@
+import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
+import React from "react";
+import {Image, StyleSheet, Text} from "react-native";
+import { Colors } from '../utils/colors';
+import CheckIn from '../screens/CheckIn';
+import Report from '../screens/Report';
+import SettingNav from './SettingNav';
+
+const Tab = createBottomTabNavigator();
+
+export default function CheckInNav(){
```

Comment:
I think the component name should be different because it contains Check-In, Report, and Setting.